.github/workflows/qit-self-group-test.yml

```diff
@@ -0,0 +1,64 @@
+name: QIT Self-Tests - Group Tests
+
+on:
+  # Every day at 11pm UTC (6pm ET)
```

Comment:
Any reason for not midnight?

Reply:
Really minor, but I know we have "nightly" crons that go at midnight so was just curious.

Reply:
Sounds good!